client/src/components/molecules/EditModal.js

```diff
@@ -152,6 +152,48 @@ const EditModal = (props) => {
             />
           </Grid>
         </Grid>
+        <Grid container spacing={2} style={{ padding: "0.5em 0"}}>
+          <Grid item xs={6}>
+            <TextFieldInput
+              id="edit_profile_picture"
+              className="edit_profile_picture"
+              label="Profile Image URL"
+              type="text"
+              placeholder="Link to image"
+              onChange={(e) =>
+                handleChange(e, "profilePicture")
+              }
+              defaultValue={props.users.user.profilePicture}
+            />
+          </Grid>
+          <Grid item xs={6}>
+            <TextFieldInput
+              id="edit_profile_contact"
+              className="edit_profile_contact"
+              label="Contact"
```

Comment:
Can we change this label to Telephone or Contact Cell, contact somewhat confused me